buildchain/buildchain/salt_tree.py

```diff
@@ -216,10 +218,12 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
 
     Path('salt/_roster/kubernetes_nodes.py'),
 
+    Path('salt/_runners/metalk8s_saltutil.py'),
+
     Path('salt/_states/containerd.py'),
     Path('salt/_states/kubeconfig.py'),
     Path('salt/_states/docker_registry.py'),
-    Path('salt/_states/kubernetes.py'),
+    Path('salt/_states/kubernetesmod.py'),
```

Comment:
Why ? Salt only renamed the kubernetes modules not the state.

Or maybe you got conflict in `_auth/kubernetes_rbac.py` but for me this one should not `import` kubernetes stuff

Reply:
Then there may still an issue in upstream Salt ;-)